packages/xpetra/src/Vector/Xpetra_TpetraVector_def.hpp

```diff
@@ -332,7 +335,10 @@ template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
     typedef EpetraNode Node;
 
 #undef XPETRA_TPETRAMULTIVECTOR_SHORT
```

Comment:
Same as above.

Reply:
I agree.  Since this has passed the PR, I'd like to get this merged though, since I think it blocks others.

We should skim the new ETI files to see what can be dropped. With substantially less stuff getting compiled in each shot, you don't need nearly as many things defined in the headers as before.